p4-16/spec/P4-16-spec.mdk

```diff
@@ -4063,6 +4082,35 @@ of the `struct` or `header` type does not need to match the order of
 the values of the structure-valued expression. All of the fields of
 the `struct` or `header` types do need to be initialized though.
 
+## Operations on list expressions { #sec-list-exprs }
+
+Literal list values can be represented as list expressions, and passed
+as arguments, e.g. to extern constructor functions.
+
+A literal list value is written using curly braces, with each element
+separated by a comma.  The left curly brace is preceded by a
+`(list<T>)` where `T` is the list element type.  This `(list<T>)` can
```

Comment:
Understood.  I was not sure whether it did, so included such a phrase in the proposed text in case it did, and figured that you would have good suggestions for improving it.  I will push another commit that removes that sentence.